src/views/4_PromptComponent.tsx

```diff
@@ -225,7 +227,7 @@ export class PromptComponent extends React.Component<Props, State> implements Ke
                          title={JSON.stringify(this.props.status)}
                          dangerouslySetInnerHTML={{__html: this.props.status === Status.Interrupted ? fontAwesome.close : ""}}></div>
                     <div className="prompt"
-                         style={css.prompt}
```

Comment:
Please convert `css.prompt` to a function and pass it `isSticky` to keep all the CSS in one place. You'll find many examples how to do it in the file.